llama_stack/providers/utils/telemetry/sqlite_trace_store.py

```diff
@@ -75,14 +75,21 @@ def build_order_clause() -> str:
                 {where_clause}
             ),
             filtered_traces AS (
-                SELECT t.trace_id, t.root_span_id, t.start_time, t.end_time
+                SELECT t.trace_id,
```

Comment:
if this PR is doing something besides mypy coverage, I recommend either splitting it into its own commit or its own PR. Please add some context to either the new PR or commit message, thanks

Reply:
will create a separate PR for this now 👍🏽

Reply:
raised https://github.com/meta-llama/llama-stack/pull/2815